pkg/defaults/defaults.go

```diff
@@ -117,7 +117,12 @@ func SetDefaultVirtualMachineInstance(clusterConfig *virtconfig.ClusterConfig, v
 	setDefaultCPUArch(clusterConfig, &vmi.Spec)
 	setGuestMemoryStatus(vmi)
 	setCurrentCPUTopologyStatus(vmi)
-	setupHotplug(clusterConfig, vmi)
+
+	// Hotplug needs to be enabled on ARM yet
+	if !IsARM64(&vmi.Spec) {
```

Comment:
didn't we have arm specific file with defaults?

Reply:
Yes we do but the hotplug is in the generic flow, this is just fix so it is temporary.

Reply:
how are we planing to follow up? I'm not sure... 
maybe we should get rid of the setupHotplug and make it arch specific?

Reply:
Follow up will be to enable hotplug for ARM. I will then test this manually since we don't have a CI.

Reply:
Got you. /approve